arches/app/models/fields/i18n.py

```diff
@@ -63,8 +63,8 @@ def as_sql(self, compiler, connection):
         https://www.postgresql.org/docs/9.5/functions-json.html
         """
 
-        if (self.value_is_primitive or self.value is None) and not isinstance(
-            compiler, SQLInsertCompiler
+        if (self.value_is_primitive or self.value is None) and isinstance(
+            compiler, SQLUpdateCompiler
```

Comment:
For a filter() call, this was coming in as just `SQLCompiler`.

Reply:
So are you saying that this should be `SQLCompiler`?